addons/hr/models/hr_department.py

```diff
@@ -135,6 +135,15 @@ def action_plan_from_department(self):
     def get_children_department_ids(self):
         return self.env['hr.department'].search([('id', 'child_of', self.ids)])
 
+    def open_view_dependent_departments(self):
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "hr.department",
+            "views": [[False, "kanban"], [False, "tree"], [False, "form"]],
+            "domain": [['id', 'in', self.get_children_department_ids().ids]],
+            "name": "Dependent departments",
```

Comment:
To be consistent
```suggestion
            "name": "Child Departments",
```

Reply:
Geoffrey ask me to replace Child Departments to Dependant departments :)

Reply:
https://www.awesomescreenshot.com/image/47449654?key=c2da6ae5eb06837714b08568775cb2de

Was this before or after :thinking:

Reply:
![image](https://github.com/odoo/odoo/assets/11148685/d1453399-d924-400a-92e0-cdee8cc17310)

Reply:
you can check on the task chatter